config/faf-clans/faf-clans.env

```diff
@@ -1,3 +1,7 @@
-WEBROOT=/usr/share/nginx/html
-TEMPLATE_NGINX_HTML=0
+API_BASE_URL=https://api.dev.faforever.com
+OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
+OAUTH_REDIRECT_URI=https://clans.faforever.com
```

Comment:
As I said; "*If you look at other .env files"*. So [if you actually do that](https://github.com/FAForever/faf-stack/blob/develop/config.template/faf-jeremy/faf-jeremy.env#L3), you'll see that they reference localhost.

How else would it work "locally"? And the clientId is a runtime configuration that is specific to the system, and we do not want any production values in the sample files. I keep repeating myself..